start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for 3pa-register-admin-web
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
```

Comment:
I think this condition is for a vagrant build, so not needed.  But perhaps leave in, and experiment taking it out once we've got it working in Docker and an e2e.